editor/editor_help.h

```diff
@@ -297,4 +299,40 @@ class EditorHelpTooltip : public EditorHelpBit {
 	EditorHelpTooltip(const String &p_text = String(), const String &p_custom_description = String());
 };
 
+#if defined(MODULE_GDSCRIPT_ENABLED) || defined(MODULE_MONO_ENABLED)
+class EditorSyntaxHighlighter;
+
+class EditorHelpHighlighter {
+public:
+	enum Language {
+		LANGUAGE_GDSCRIPT,
+		LANGUAGE_CSHARP,
+		LANGUAGE_MAX,
+	};
```

Comment:
Ideally we should find an abstraction here that avoids coupling EditorHelp to specific modules, and enables third-party script language integrations to add highlighting to their own docs (e.g. Rust, Swift, etc.).

That could be left for future work though.

Reply:
This feature already hardcodes GDScript + C#, so I did not introduce abstractions that are currently useless.